src/context/example.rs

```diff
@@ -0,0 +1,53 @@
+use report::example::ExampleReport;
+
+#[derive(Clone, Copy, PartialEq, Eq, Debug)]
+pub enum ExampleLabel {
```

Comment:
But is getting rid of 9 clear and readable lines …

```rust
impl From<ExampleLabel> for &'static str {
    fn from(label: ExampleLabel) -> Self {
        match label {
            ExampleLabel::It => "It",
            ExampleLabel::Example => "Example",
            ExampleLabel::Then => "Then",
        }
    }
}
```

… worth the additional dependency and macro magic? With the above you know what's going on. With a macro one would have to it look up.